languages/go/exercises/concept/conditionals/.docs/after.md

```diff
@@ -0,0 +1 @@
+# After
```

Comment:
If the document is empty, I'd remove it.

Reply:
updated!